orca-front50/src/test/groovy/com/netflix/spinnaker/orca/front50/pipeline/PipelineStageSpec.groovy

```diff
@@ -33,9 +32,7 @@ class PipelineStageSpec extends Specification {
   @Unroll
   def "should cancel child pipeline (if started)"() {
     given:
-    def stage = new com.netflix.spinnaker.orca.pipeline.model.PipelineStage(
-      new Pipeline(), "pipeline", stageContext
-    )
```

Comment:
No longer a name clash with the "other" `PipelineStage` (a stage that runs a pipeline).